### DISCUSSION TITLE
使用RPC Filter技术防御内网RPC横向攻击 « 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#11] [General]
# 使用RPC Filter技术防御内网RPC横向攻击 « 倾旋的博客

倾旋的博客

https://payloads.online/archivers/2022-10-23/1/

**Comment by lawleetSkc** [comment]
倾选哥，关注你很久了，我也是专攻内网攻防研究的，不过这种一棍子打死的策略在流量侧似乎有点极端了，因为我们从IDS侧反馈回来的告警，有大量合法的用户业务流量也会使用这些RPC，例如IOXID网卡扫描，SRVSVC，内网这一块只能作为可疑行为，或许使用日志记录，在溯源中回调才是有价值的用途，这些RPC需要行为链的event剧本去分析和定性。

**Reply by lawleetSkc** [reply]
所以从终端角度使用RPC filter或许也可能阻断正常业务

**Reply by Rvn0xsy** [reply]
是的，需要酌情考虑，但是个人终端我觉得可以上策略试试

**Comment by rantrism** [comment]
您好～我是腾讯云开发者社区的运营，关注了您分享的技术博客，觉得内容很棒，我们诚挚邀请您加入腾讯云自媒体分享计划。完整福利和申请地址请见：https://cloud.tencent.com/developer/support-plan 作者申请此计划后将作者的文章进行搬迁同步到社区的专栏下，你只需要简单填写一下表单申请即可，我们会给作者提供包括流量、云服务器等，另外还有些周边礼物。

**Comment by bcdlbgm** [comment]
在针对2016版本中使用atexec进行攻击，协议只有smb3，不同的版本中使用的协议也不相同。

**Comment by xiaoheiccc** [comment]
at从windows8以后已启用，后续用的都是schtasks哇
